### DISCUSSION TITLE
2024-04-19(10年記念)更新

### DISCUSSION THREAD

**Discussion by CSReviser** [CSReviser/CaptureStream2#14] [Announcements]
2024-04-19(10年記念)更新

最終更新（2014/04/19）から10年

<hr /><em>This discussion was created from the release <a href='https://github.com/CSReviser/CaptureStream2/releases/tag/20240419'>2024-04-19(10年記念)更新</a>.</em>

**Comment by Kouzou91** [comment]
ubuntu 22.04 を使っていますが、
$ export DISPLAY=:0 && $HOME/LinCaptureStream2/CaptureStream2 -nogui
とすると segmentation error が出ます。
crontab -e から cron に登録してみても上手く動作しないようです。
但し、windows版を wine で使うと上手く動作します。
宜しくご検討下さい。

**Reply by CSReviser** [reply]
動作報告くださりありがとうございます。
当方の環境では残念ながら不具合現象が確認できていません。
（１）具体的、正確なエラメッセージをお知らせください。
（２）今回のバージョンが初めて使用するバージョンでしょうか？過去のバージョンでは使用できていて、今回バージョンアップして不具合となったのでしょうか？過去のバージョンは使えていたという場合はそのバージョンをお知らせください。
（３）ubuntu 22.04にインストールされているQt6のバージョンをお知らせください。
（４）GUI（-noguiオプションなし）では使用できているのでしょうか？
（５）使用されているPCの搭載メモリは何GBでしょうか？

問題解決できるかどうかわかりませんが、情報提供をお願いいたします。

**Reply by CSReviser** [reply]
https://csreviser.github.io/CaptureStream2/install_linux
Ubuntu版のバイナリ（実行ファイル）ビルド手順を作成しました。参考にしてお試しください。
こちらのサイトで提供しているバイナリ（実行ファイル）が環境によって動作しない可能性が考えられます。
ご自身の環境でビルドすることにより、動作することを期待します。
